README.md

```diff
@@ -78,8 +78,8 @@ export default class App extends Component {
       })
     }
     
-    const search = (tag, input, selectedTags) => {
-      tag.label.toLowerCase().includes(input.toLowerCase()) && !selected.includes(tag)
+    const search = (tag, input) => {
+      tag.label.toLowerCase().includes(input.toLowerCase())
```

Comment:
Don't you need to add the search prop to the component in the `return` statement below?

```javascript
<TagBox
  search={search}
```